src/main/webapp/features/iq/charting/iqChart.js

```diff
@@ -26,24 +26,26 @@ const ChartColor = {
   MODERATE : '#f5c648'
 };
 
+const policyEvaluations = JSON.parse(document.currentScript.getAttribute('policyEvaluations'));
```

Comment:
0 : do we need to guard this and provide a default if the parse fails?

Reply:
it shouldn't happen but of course I'll surround it with try-catch block.